### ISSUE TITLE
Crash (Access Violation) when Importing ~135 Graphical Resources

### ISSUE THREAD

**Issue by jtwine** [godotengine/godot#80318]
### Godot version

v4.1.1.stable.official [bd6af8e0e]

### System information

Godot v4.1.1.stable - Windows 10.0.22621 - Vulkan (Compatibility) - NVIDIA GeForce GTX 1060 (NVIDIA; 30.0.14.7311) - Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 Threads)

### Issue description

_(I have seen some other existing bugs related to import, mostly on MacOS, but nothing that looks exactly like what I can reproduce here.)_

Godot will crash when trying to import ~135 graphical resources at once.  (I have not tried to nail down the exact number that causes the error.)  This occurs regardless of the number of subdirectories involved - it seems to involve the number of assets themselves.  Also, once this crash occurs via the recreation scenario below, you will _keep_ crashing Godot trying to open the same project unless you manually remove the assets from the project's filesystem.

Crash appears to be an **Access Violation** (Exception `0xC0000005`).  The WER file is attached.  
[Godot.Report.txt](https://github.com/godotengine/godot/files/12268593/Godot.Report.txt)

### Steps to reproduce

1. Launch Godot and create a new, empty project.
2. Download the code/project archive from [https://github.com/clear-code-projects/UltimateGodotIntro2D](https://github.com/clear-code-projects/UltimateGodotIntro2D), open it, and open the **resources** subdirectory in the archive
4. Open Explorer and navigate to the location of the newly created Godot project (filesystem location)
5. Drag and drop the **graphics** directory from the archive to the new project's _filesystem_ location
6. Activate the Godot window

Godot will start importing the graphical assets once it has been activated, and will crash toward the end of the import process.  If you restart Godot and attempt to reopen that the same project, import will start again automatically and will result in another crash. 

Note that if you extract all of the graphical assets and put them into a single directory (which yields 135 files), and then copy that directory to the project's _filesystem_ location, it will also crash.

The crash will not occur if you import a smaller number of files at at time.  For example, if you descend into the **graphics** directory you will have 12 subdirectories.  If you copy them over and let them complete the import process **one subdirectory at a time**, the crash will not occur, so it does not appear to be a problem with a _specific file_ being imported. 

### Minimal reproduction project

N/A

**Comment by rsubtil** [comment]
Can you test if this occurs with [v4.1](https://github.com/godotengine/godot/releases/tag/4.1-stable) and [v4.0.4](https://github.com/godotengine/godot/releases/tag/4.0.4-stable) as well?

**Comment by jtwine** [comment]
> Can you test if this occurs with [v4.1](https://github.com/godotengine/godot/releases/tag/4.1-stable) and [v4.0.4](https://github.com/godotengine/godot/releases/tag/4.0.4-stable) as well?

Yep - crashes with both **Godot_v4.0.4-stable_win64.exe** and **Godot_v4.1-stable_win64.exe** obtained from the links you provided.

If anyone is unable to get to the link I provided above, I can put a ZIP file of the graphics resources somewhere else.

-=- James.

**Comment by jsjtxietian** [comment]
I test it with 4.2.dev, it freezes for a few times but didnt crash.

**Comment by jtwine** [comment]
> I test it with 4.2.dev, it freezes for a few times but didnt crash.

Can you give me a link to the **_exact_** version you tried it with, and I will try it here just in case the user's hardware plays a part?

Thanks!

Peace!

-=- James.

**Comment by jtwine** [comment]
> @jtwine I'm using Godot v4.2.dev ([a0f5724]...

Sorry, I misread that you were using `4.2.dev` - I thought it was with an _older_ version, not the yet-to-be-released one.  My bad.

-=- James.
